src/components/MultiHopTrade/components/RateGasRow.tsx

```diff
@@ -52,56 +34,20 @@ const rateHover = {
 
 export const RateGasRow: FC<RateGasRowProps> = memo(
   ({
-    sellSymbol,
-    buySymbol,
-    rate,
-    gasFee,
+    buyAssetSymbol,
+    children,
+    isDisabled,
     isLoading,
-    allowSelectQuote,
+    rate,
+    sellAssetSymbol,
     swapperName,
     swapSource,
-    onRateClick,
-    children,
+    totalNetworkFeeFiatPrecision,
+    onClick,
   }) => {
     const translate = useTranslate()
     const { isOpen, onToggle } = useDisclosure()
 
-    const swapperIcons = useMemo(() => {
-      const isStreaming =
-        swapSource === THORCHAIN_STREAM_SWAP_SOURCE ||
-        swapSource === THORCHAIN_LONGTAIL_STREAMING_SWAP_SOURCE
-      return (
-        <AnimatePresence>
-          {isStreaming && (
-            <SlideTransitionX key={swapSource ?? swapperName}>
-              <Center
-                className='quote-icon'
-                bg='background.surface.raised.base'
-                borderRadius='md'
-                borderWidth={1}
-                p='2px'
-                borderColor='border.base'
-                boxShadow='0 1px 2px rgba(0,0,0,.2)'
-              >
-                <StreamIcon color='text.success' />
-              </Center>
-            </SlideTransitionX>
-          )}
-          <Center
-            className='quote-icon'
-            bg='background.surface.raised.base'
-            borderRadius='md'
-            borderWidth={1}
-            p='2px'
-            borderColor='border.base'
-            boxShadow='0 1px 2px rgba(0,0,0,.2)'
-          >
-            {swapperName && <SwapperIcon size='2xs' swapperName={swapperName} />}
-          </Center>
-        </AnimatePresence>
-      )
-    }, [swapSource, swapperName])
-
```

Comment:
Was moved into a separate component `SwapperIcons` during a different refactor and decided to leave it separated.